packages/core/src/App/Containers/RealAccountSignup/financial-details-form.js

```diff
@@ -0,0 +1,410 @@
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    other_instruments_trading_experience: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    other_instruments_trading_frequency: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    source_of_wealth: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [],
+    },
+};
+
+export const financialDetailsConfig = ({ real_account_signup_target }) => {
+    return {
+        header: {
+            active_title: localize('Complete your financial assessment'),
+            title: localize('Financial assessment'),
+        },
+        body: FinancialDetails,
+        form_value: getDefaultFields(real_account_signup_target, financial_details_config),
+        props: {
+            validate: generateValidationFunction(real_account_signup_target, financial_details_config),
+            account_turnover_enum: account_turnover_enum(),
+            binary_options_trading_experience_enum: binary_options_trading_experience_enum(),
+            binary_options_trading_frequency_enum: binary_options_trading_frequency_enum(),
+            cfd_trading_experience_enum: cfd_trading_experience_enum(),
+            cfd_trading_frequency_enum: cfd_trading_frequency_enum(),
+            education_level_enum: education_level_enum(),
+            employment_industry_enum: employment_industry_enum(),
+            employment_status_enum: employment_status_enum(),
+            forex_trading_experience_enum: forex_trading_experience_enum(),
+            forex_trading_frequency_enum: forex_trading_frequency_enum(),
+            estimated_worth_enum: estimated_worth_enum(),
+            income_source_enum: income_source_enum(),
+            net_income_enum: net_income_enum(),
+            occupation_enum: occupation_enum(),
+            other_instruments_trading_experience_enum: other_instruments_trading_experience_enum(),
+            other_instruments_trading_frequency_enum: other_instruments_trading_frequency_enum(),
+            source_of_wealth_enum: source_of_wealth_enum(),
+        },
+        passthrough: ['residence_list', 'is_fully_authenticated'],
+    };
+};
+const account_turnover_enum = () => [
+    {
+        value: 'Less than $25,000',
+        text: localize('Less than $25,000'),
+    },
+    {
+        value: '$25,000 - $50,000',
+        text: localize('$25,000 - $50,000'),
+    },
+    {
+        value: '$50,001 - $100,000',
+        text: localize('$50,001 - $100,000'),
+    },
+    {
+        value: '$100,001 - $500,000',
+        text: localize('$100,001 - $500,000'),
+    },
+    {
+        value: 'Over $500,000',
+        text: localize('Over $500,000'),
+    },
+];
+const binary_options_trading_experience_enum = () => [
+    {
+        value: '0-1 year',
+        text: localize('0-1 year'),
```

Comment:
Also, when we `localize` strings like this in constant files, we risk loading them before the i18n service has been initialised. It would then only render the correct localised string when `localize` has been called again here through a re-render of the `binary_options_trading_experience_enum` invocation. Are we sure these are loaded after the i18n service?

Reply:
These functions are called on Account wizard's `componentDidMount` method. by that time, I think `i18n` is already initialized.